bin/pqcheck

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bin/ruby
```

Comment:
I think this should invoke Logstash's vendored jruby, right?

Reply:
`bin/ruby` is the LS bootstrapping code to execute a "plain" ruby script.